operator/v1/types_ingress.go

```diff
@@ -1038,19 +1032,17 @@ type IngressControllerHTTPHeaderBuffer struct {
 	// +kubebuilder:validation:Minimum=4096
 	// +optional
 	HeaderBufferMaxRewriteBytes int32 `json:"headerBufferMaxRewriteBytes,omitempty"`
-}
-
-type IngressControllerThreading struct {
-	// count defines the number of threads created per ingress controller pod.
-	// Creating more threads allows each ingress controller pod to handle more
-	// connections, at the cost of more system resources being used. If this
-	// field is empty, the IngressController will use the default value. The
-	// current default is 4 threads, but this may change in future releases.
+	// threadCount defines the number of threads created per ingress controller
```

Comment:
We don't typically mention "haproxy" but I wonder whether we should here. This configures the number of threads in the haproxy process, not in a "ingress controller pod". Should we call out haproxy by name?

Reply:
I think you're right, it's an important detail that it's more threads of HAProxy, not openshift-router or anything else. I've updated it in the latest commit